dashboard/lib/policies/lti.rb

```diff
@@ -131,6 +131,10 @@ def self.early_access_closed?
     LtiIntegration.count >= lti_early_access_limit
   end
 
+  def self.early_access_banner_available?(user)
+    early_access? && lti?(user)
```

Comment:
I clarified with @alivira and the banner should only be displayed for teachers.
```suggestion
    early_access? && lti?(user) && user.teacher?
```